provisioning/src/handlers/portfolioDrafts/post.ts

```diff
@@ -0,0 +1,31 @@
+import { Context, HttpRequest } from '@azure/functions'
+const { CosmosClient } = require("@azure/cosmos");
+import { v4 as uuidv4 } from 'uuid';
+import dayjs from "dayjs";
+import {PortfolioSummary} from "../../lib/models/PortfolioSummary";
+import {ProvisioningStatus} from "../../lib/models/ProvisioningStatus";
+
+export async function handler (context: Context, req: HttpRequest): Promise<void> {
```

Comment:
would this function break if we remove the req param? it is not being used

Reply:
Nope we should be able to remove it.